.travis.yml

```diff
@@ -5,13 +5,14 @@ matrix:
       env: TOX_ENV=pep8
     - python: "3.4"
       env: TOX_ENV=py34
-before_install:
-  - sudo apt-get -qq update
-  - sudo apt-get install -y build-essential libcap-dev
```

Comment:
I don't think we need these lines. Maybe we do? Everything seems to work fine without them. I am guessing they were cargo-culted in from another project?